tests/util/test_logging_filter.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+import logging
+from re import split
+from time import sleep
+
+import pytest
+
+from chia.util.logging import TimedDuplicateFilter
+
+
+def test_logging_filter(caplog: pytest.LogCaptureFixture) -> None:
+
+    log_interval_secs = 10
+    num_logs = 11
```

Comment:
I think it would make sense to use as small numbers as possible here, to make the test run as quickly as possible. e.g. set the limit to 1 second and sleep for 95ms.

Reply:
Good idea.